src/QsCompiler/Transformations/ClassicallyControlledTransformation.cs

```diff
@@ -186,56 +232,53 @@ private TypedExpression GetApplyIfExpression(QsResult result, TypedExpression co
                         if (adj && ctl)
                         {
                             controlOpInfo = BuiltIn.ApplyIfElseCA;
```

Comment:
I think you are missing an R here: 
```suggestion
                            controlOpInfo = BuiltIn.ApplyIfElseRCA;
```